1-Draft/RFCXXXX-Policy.md

```diff
@@ -0,0 +1,234 @@
+
+The default values must be `secure-by-default`.
+
+For release versions hard-coded defaults must be the same as ones in re-installed configuration files. For preview versions they may vary (ex., enable experimental features and so on).
+
+If during startup PowerShell 7 cannot read system configuration files it fails to startup.
+
+If during startup PowerShell 7 cannot read user configuration files it uses _hardcoded_ defaults.
+
+If during operation PowerShell 7 cannot read configuration files it continue to use _current_ (runtime) configuration values.
+
+### Settings locations
+
+`PowerShell 7` settings are grouped into `Policy settings` and `Regular settings`.
+Regular settings are normal configuration settings.
+Regular settings can be treated as default values.
+Policy settings is high priority and overlap regular settings.
+Policy settings are used by administrators to centrally manage applications.
+
+| Location     | Policy settings                                           | Regular settings                                           |
+|--------------|-----------------------------------------------------------|------------------------------------------------------------|
+| File section | "PowerShell": { "PolicySettings": {...} }                 | "PowerShell": { "RegularSettings": {...} }                 |
+| File section | "OtherPowerShellApplication": { "PolicySettings": {...} } | "OtherPowerShellApplication": { "RegularSettings": {...} } |
+| Registry key | Software\Policies\PowerShellCore                          | Software\PowerShellCore                                    |
+
+### Policy settings Setting Fall-Back
+
+#### Motivation - Policy Setting Fall-Back
+
+This is to allow Fall-back to Windows PowerShell policies.
+
+#### Implementation
+
+For Policy Settings,
+each policy should have a `Use Windows PowerShell Policy` which will indicate that the policy should the read from
+`SOFTWARE\Policies\Microsoft\Windows\PowerShell` instead of `Software\Policies\PowerShellCore`.
+
+### Precedence of applying settings
+
+Because a configuration setting can be in several schemes, the setting wins according to the priority of its scheme.
+
+#### Precedence for Policy settings in descending order
+
+| Scheme                      | Windows                                              | Unix                                                 |
+|-----------------------------|------------------------------------------------------|------------------------------------------------------|
+| GPO -> Computer Policy      | HKLM\Software\Policies\PowerShellCore                | /etc/powershell.config.json                          |
+| GPO -> User Policy          | HKCU\Software\Policies\PowerShellCore                | See `Comment A` below                                |
+| File -> Computer-Wide       | %ProgramFiles%/PowerShell/powershell.config.json     | /opt/Microsoft/powershell/powershell.config.json     |
```

Comment:
Why do we use "Microsoft" in the path?

Reply:
This should be in a `/etc` path.  I think this and the one on line 88 should be in `/etc/powershell/powershell.config.json`

Reply:
```suggestion
| File -> Computer-Wide       | %ProgramFiles%/PowerShell/powershell.config.json     | /etc/powershell/powershell.config.json     |
```